README.md

```diff
@@ -354,12 +325,6 @@ are created at a system-wide level, ignoring the annotations.
 
 Using these flags and annotations, you can run the CCM on a node in a different metro or facility, or even outside of Equinix Metal entirely.
 
-#### Control Plane LoadBalancer Implementation
-
-For the control plane nodes, the Equinix Metal CCM uses static Elastic IP assignment, via the Equinix Metal API, to tell the
-Equinix Metal network which control plane node should receive the traffic. For more details on the control plane
-load-balancer, see [this section](#control-plane-load-balancing).
-
 #### Service LoadBalancer Implementations
```

Comment:
We should link to this section above in the table where the `loadbalancer` setting is introduced without any options described.

https://github.com/kubernetes-sigs/cloud-provider-equinix-metal/pull/470/commits/8a048dc5b3f9b735b49877bf41b8f15f265dfc7c#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R224